packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -94,6 +94,54 @@ export enum EcsMetric {
    * GPUReservation Metric
    */
   GPU_RESERVATION = 'GPUReservation',
+  /**
+   * ActiveConnectionCount Metric
```

Comment:
Please add some more detail here. Customers should be able to tell that these are all Service Connect metrics from the comments, and possibly we should include a plain english description of the metric when it's not immediately clear from the name.

Reply:
we might need to fix "* Units of this metrics are percentages" too

Reply:
After a second thought i feel like we probably don’t need to define `EcsMetric` as everytime ecs adds a new metric we’ll have to update the list 🤔 For example, EBS will introduce its own metrics. 

Instead, we just need to keep a slice of string for what metrics are only available when SC is enabled, which is [this](https://github.com/homakk/aws-cdk/pull/4/files#diff-becce6466790cd3cc81807ab64dd5f4ef85eed0285509d1ae43b381f24aefddaR1656) and we can make it a map or simply a slice using include for validation. That list i assume won’t grow too much as the feature is already out. And it would make more sense for us to maintain that 💭